Documentation/Diagnostics/PH2159.md

```diff
@@ -0,0 +1,139 @@
+# PH2159: TestTimeouts class should be internal
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MsTestAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MsTestAnalyzers) |
+| Diagnostic ID | PH2159 |
+| Category  | [MsTest](../MsTest.md) |
+| Analyzer | [TestTimeoutsClassAccessibilityAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/main/Philips.CodeAnalysis.MsTestAnalyzers/TestTimeoutsClassAccessibilityAnalyzer.cs) |
+| CodeFix  | Yes |
+| Severity | Warning |
+| Enabled By Default | Yes |
+
+## Introduction
+
+Projects commonly define a `TestTimeouts` class that centralizes timeout constants (e.g., `CiAppropriate`, `Integration`). This analyzer enforces that any class named `TestTimeouts` is declared as `internal` and not `static`. Public exposure or use of `static` on this coordination class is discouraged.

```

Comment:
Perhaps mention how this rule helps avoid conflict with the MSTEST analyzer that says all classes in a test project cannot be public unless it's a [TestClass]